tools/cli/admin_elastic_search_commands.go

```diff
@@ -390,7 +392,7 @@ func GenerateReport(c *cli.Context) error {
 	}
 	buckets = groupby["buckets"].([]interface{})
 	if len(buckets) == 0 {
-		fmt.Println("no matching bucket")
+		output.Write([]byte("no matching bucket"))
```

Comment:
let's add `\n` at the end of the string to match with previous behavior which was calling `Println`